packages/driver/src/dom/dimensions.ts

```diff
@@ -68,15 +94,15 @@ const getNumAttrValue = ($el, attr) => {
   return num
 }
 
-const getPadding = ($el, dir) => {
+const getPadding = ($el, dir: dir) => {
```

Comment:
At a minimum we probably should explicitly type `$el` as `any` in these functions similar to above? Do we have any idea what the expected type is, like `JQuery<HTMLElement>`?

Reply:
Addressed in https://github.com/cypress-io/cypress/pull/31003/commits/cba0cc3cacc14d00bd28862f95a6ebd74d90235b